neat_core.c

```diff
@@ -30,6 +33,9 @@
 #endif
 #ifdef __linux__
     #include "neat_linux_internal.h"
+#if !defined(HAVE_NETINET_SCTP_H) && !defined(USRSCTP_SUPPORT)
+    #include <linux/sctp.h>
```

Comment:
Was missing that header, was not aware of needing the libsctp-dev package to use a kernel feature. Adding a note to README about that, as well.